src/Mvc/Mvc.ApiExplorer/src/EndpointMetadataApiDescriptionProvider.cs

```diff
@@ -68,6 +69,17 @@ public void OnProvidersExecuted(ApiDescriptionProviderContext context)
 
     private ApiDescription CreateApiDescription(RouteEndpoint routeEndpoint, string httpMethod, MethodInfo methodInfo)
     {
+        static bool ShouldDisableInferredBody(string method)
```

Comment:
More copied logic between Map and the api description provider 😢

Reply:
Can we move this to shared source to prevent these from going out of sync? Maybe rename `ParameterBindingMethodCache` to be more general. If not, I'd say just add a comment to the `ShouldDisableInferredBody(string method)` in `EndpointRouteBuilderExtensions` to keep these in sync.

Reply:
Oh, I just noticed that this should be called against all the methods in `IHttpMethodMetadata.HttpMethods`, not just the one we're creating the ApiDescription for currently to align with the logic in `EndpointRouteBuilderExtensions`.